api/v1/canary_types.go

```diff
@@ -203,11 +213,10 @@ type CanaryStatus struct {
 // Canary is the Schema for the canaries API
 // +kubebuilder:printcolumn:name="Interval",type=string,JSONPath=`.spec.interval`
 // +kubebuilder:printcolumn:name="Status",type=string,JSONPath=`.status.status`
-// +kubebuilder:printcolumn:name="Message",type=string,JSONPath=`.status.message`
+// +kubebuilder:printcolumn:name="Last Check",type=date,JSONPath=`.status.lastCheck`
```

Comment:
I think we still want the (first/error) message ?

Reply:
I think we take the first non-empty message/error and then add `...` if there are more